changelog/_unreleased/2024-11-29-refactored-internals-of-the-mailservice.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Refactored internals of the MailService
+author: Max
+author_email: max@swk-web.com
+author_github: @aragon999
+---
+# Core
+* Changed internals of the `Shopware\Core\Content\Mail\Service\MailService`
```

Comment:
if we do not state why we did that, it can also be omitted.

Reply:
I added some more text, why I did it, or should that be removed completely?

Reply:
looks good :+1: